providers/os/resources/packages/windows_packages.go

```diff
@@ -278,9 +280,63 @@ func (w *WinPkgManager) getLocalInstalledApps() ([]Package, error) {
 			packages = append(packages, *p)
 		}
 	}
+
+	// These are the .NET Framework packages
+	// They do not show up in the general apps or features list, so we need to discover them separately
+	dotNetFramework, err := w.getDotNetFramework()
+	if err != nil {
+		return nil, err
+	}
+	packages = append(packages, dotNetFramework...)
+
 	return packages, nil
 }
 
+// getDotNetFramework returns the .NET Framework package
+func (w *WinPkgManager) getDotNetFramework() ([]Package, error) {
+	// https://learn.microsoft.com/en-us/dotnet/framework/install/how-to-determine-which-versions-are-installed#net-framework-45-and-later-versions
+	dotNet45plus := "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\NET Framework Setup\\NDP\\v4\\Full"
+	// https://learn.microsoft.com/en-us/dotnet/framework/install/how-to-determine-which-versions-are-installed#use-registry-editor-older-framework-versions
+	dotNet35 := "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\NET Framework Setup\\NDP\\v3.5"
+
+	return getDotNetFrameworkPackageFromRegistryKeys(dotNet45plus, dotNet35, w.platform)
+}
+
+// getDotNetFrameworkFs returns the .NET Framework package discovered on the filesystem
+func (w *WinPkgManager) getDotNetFrameworkFs() ([]Package, error) {
+	// https://learn.microsoft.com/en-us/dotnet/framework/install/how-to-determine-which-versions-are-installed#net-framework-45-and-later-versions
+	dotNet45plus := "Microsoft\\NET Framework Setup\\NDP\\v4\\Full"
+	// https://learn.microsoft.com/en-us/dotnet/framework/install/how-to-determine-which-versions-are-installed#use-registry-editor-older-framework-versions
+	dotNet35 := "Microsoft\\NET Framework Setup\\NDP\\v3.5"
+
+	return getDotNetFrameworkPackageFromRegistryKeys(dotNet45plus, dotNet35, w.platform)
```

Comment:
is this the connection platform or the local one? e.g. for fs scanning, would this be the detected platform or the one where we're running the scanning from

Reply:
I don't know.
I used the same platform that we also use for the other packages.